yunhyunglee/utils.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+def VALIDATOR_EMAIL(email):
```

Comment:
```suggestion
def VALIDATE_EMAIL(email):
```
진짜 하나도 안중요하다고 생각할 수 있는데. VALIDATOR라고 하면 EMAIL이  앞으로 가야 할것 같고 아니면 저렇게 하는게 나을것 같네요. 관련 코드 수정 부탁드립니다.

Reply:
넵 수정하겠습니다 컨플릭트는 브랜치에서 메인으로 가지 않고 바로 다른 브랜치로 옮겨 작업해서 나는것 같은데 어떻게 수정하면 될까요?